### DISCUSSION TITLE
Question - How to override utilities in runtime

### DISCUSSION THREAD

**Discussion by liaoliao666** [jaredh159/tailwind-react-native-classnames#216] [Ideas]
My goal is to change the font size at runtime as follows

```js
tw.setUtilities("text-content", 'text-[17px]')
```

**Comment by jaredh159** [comment]
There's no current way to do something like this. Although there's nothing stopping you from providing runtime arbitrary values, like:

```tsx
<View style={tw`text-[${someVar}]`} />
```

Would that satisfy your use case?

**Comment by liaoliao666** [comment]
This way is so inconvinient when u got a lot of utilities u want update.
But in the web, this is easily achieved with css variables like this
```js
.text-content: {
  color: `var(--text-content)`
}
```
And it seems difficult to implement in React Native.


Or Maybe this library can support a way to allow user-defined prefixes. like below a user-defined theme `dim`
```js
tw`text-[#333] dark:text-[#fff]  dim:text-[#e7e9ea]`
```

**Comment by jaredh159** [comment]
are you targeting RN _Web_?

**Comment by liaoliao666** [comment]
No. just show an example how can i update tailwind uitility in web. but can not do this way in RN.
